core/valor_core/classification.py

```diff
@@ -0,0 +1,1343 @@
+import time
+from collections import defaultdict
+from typing import Dict, List, Optional, Tuple, Union
+
+import numpy as np
+import pandas as pd
+from valor_core import enums, metrics, schemas, utilities
+
+
+def _create_classification_grouper_mappings(
```

Comment:
Are we switching back to label mapping within the computation?

I was under the impression we were happy with it residing in the data aggregation step since the computation is unnecessarily complicated by it.

Reply:
The data aggregation approach replaces the label  id, key, value in-place so the computation never has to care what the label was originally.

Reply:
yes, the core reason being that pandas requires me to drop the mapping columns during some `group_by` operations, so I need a way to map them back after the `group_by` is done.

Reply:
I can work on this. the data aggregation change happened after I'd already finished this part of the code

Reply:
made this change. thanks for the suggestion!